diff --git a/targets/PROJECTS/GENERIC-NR-5GC/CONF/du_gnb.conf b/targets/PROJECTS/GENERIC-NR-5GC/CONF/du_gnb.conf
index f3c95f1bc7..87f0760f47 100644
--- a/targets/PROJECTS/GENERIC-NR-5GC/CONF/du_gnb.conf
+++ b/targets/PROJECTS/GENERIC-NR-5GC/CONF/du_gnb.conf
@@ -15,7 +15,7 @@ gNBs =
 
     // Tracking area code, 0x0000 and 0xfffe are reserved values
     tracking_area_code  =  1;
-    plmn_list = ({ mcc = 208; mnc = 99; mnc_length = 2; snssaiList = ({ sst = 1 }) });
+    plmn_list = ({ mcc = 001; mnc = 01; mnc_length = 2; snssaiList = ({ sst = 1 }) });
 
     nr_cellid = 12345678L;
 
@@ -169,8 +169,8 @@ MACRLCs = (
     tr_s_preference  = "local_L1";
     tr_n_preference  = "f1";
     local_n_if_name = "lo";
-    local_n_address = "127.0.0.3";
-    remote_n_address = "127.0.0.4";
+    local_n_address = "192.168.1.56";
+    remote_n_address = "192.168.1.233";
     local_n_portc   = 500;
     local_n_portd   = 2152;
     remote_n_portc  = 501;

