diff --git a/targets/PROJECTS/GENERIC-NR-5GC/CONF/cu_gnb.conf b/targets/PROJECTS/GENERIC-NR-5GC/CONF/cu_gnb.conf
index 493680900c..de7ca41f49 100644
--- a/targets/PROJECTS/GENERIC-NR-5GC/CONF/cu_gnb.conf
+++ b/targets/PROJECTS/GENERIC-NR-5GC/CONF/cu_gnb.conf
@@ -15,15 +15,15 @@ gNBs =
 
     // Tracking area code, 0x0000 and 0xfffe are reserved values
     tracking_area_code  =  1;
-    plmn_list = ({ mcc = 208; mnc = 99; mnc_length = 2; snssaiList = ({ sst = 1 }) });
+    plmn_list = ({ mcc = 001; mnc = 01; mnc_length = 2; snssaiList = ({ sst = 1 }) });
 
     nr_cellid = 12345678L;
 
     tr_s_preference = "f1";
 
     local_s_if_name = "lo";
-    local_s_address = "127.0.0.4";
-    remote_s_address = "127.0.0.3";
+    local_s_address = "192.168.1.233";
+    remote_s_address = "192.168.1.56";
     local_s_portc   = 501;
     local_s_portd   = 2152;
     remote_s_portc  = 500;
@@ -39,7 +39,7 @@ gNBs =
 
 
     ////////// AMF parameters:
-        amf_ip_address      = ( { ipv4       = "192.168.69.131";
+        amf_ip_address      = ( { ipv4       = "192.168.70.132";
                               ipv6       = "192:168:30::17";
                               active     = "yes";
                               preference = "ipv4";
@@ -50,9 +50,9 @@ gNBs =
     {
 
         GNB_INTERFACE_NAME_FOR_NG_AMF            = "eth0";
-        GNB_IPV4_ADDRESS_FOR_NG_AMF              = "192.168.18.207";
+        GNB_IPV4_ADDRESS_FOR_NG_AMF              = "192.168.70.129";
         GNB_INTERFACE_NAME_FOR_NGU               = "eth0";
-        GNB_IPV4_ADDRESS_FOR_NGU                 = "192.168.18.207";
+        GNB_IPV4_ADDRESS_FOR_NGU                 = "192.168.70.129";
         GNB_PORT_FOR_S1U                         = 2152; # Spec 2152
     };
   }

